src/main/resources/add-ons-to-index.json

```diff
@@ -499,6 +514,9 @@
       "uid": "org.openmrs.module.conceptmanagementapps",
       "type": "OMOD",
       "name": "Concept Management Apps Module",
+       "tags": [
+        "App", "Concept", "app-management"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.

Reply:
app-management still seems fine but the others do not really fit in